warehouse/ingest-core/src/main/java/datawave/ingest/mapreduce/job/BulkIngestMapFileLoader.java

```diff
@@ -1133,41 +1134,43 @@ private void writeStats(Path[] jobDirectories) throws IOException {
             // Write out the metrics.
             // We are going to serialize the counters into a file in HDFS.
             // The context was set in the processKeyValues method below, and should not be null. We'll guard against NPE anyway
-            FileSystem fs = getFileSystem(seqFileHdfs);
-            RawLocalFileSystem rawFS = new RawLocalFileSystem();
-            rawFS.setConf(conf);
-            CompressionCodec cc = new GzipCodec();
-            CompressionType ct = CompressionType.BLOCK;
-            
-            Counters c = reporter.getCounters();
-            if (null != c && c.countCounters() > 0) {
-                // Serialize the counters to a file in HDFS.
-                Path src = new Path(File.createTempFile("MapFileLoader", ".metrics").getAbsolutePath());
-                Writer writer = SequenceFile.createWriter(conf, Writer.file(rawFS.makeQualified(src)), Writer.keyClass(NullWritable.class),
-                                Writer.valueClass(Counters.class), Writer.compression(ct, cc));
-                writer.append(NullWritable.get(), c);
-                writer.close();
+            try (FileSystem fs = getFileSystem(seqFileHdfs); RawLocalFileSystem rawFS = new RawLocalFileSystem()) {
```

Comment:
As I had commented before, I do not believe we want to be closing FileSystem objects.